examples/all-clusters-app/all-clusters-common/src/dishwasher-mode.cpp

```diff
@@ -76,12 +76,23 @@ CHIP_ERROR DishwasherModeDelegate::GetModeTagsByIndex(uint8_t modeIndex, List<Mo
     return CHIP_NO_ERROR;
 }
 
+void DishwasherMode::Shutdown()
+{
+    if (gDishwasherModeInstance != nullptr)
+    {
+        gDishwasherModeInstance->~Instance();
```

Comment:
Please make this symmetric with the allocator.  So:
```suggestion
        delete gDishwasherModeInstance;
        gDishwasherModeInstance = nullptr;
```
This will (1) properly free the memory and (2) be safe if Shutdown is called twice (e.g. after a second failed init or something).  Similar elsewhere.

Reply:
Done.